### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPF Message Box_ScreenReader: Screen reader is not announcing Expanded State for combobox.

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1459]
Copied from VSO [951013](https://devdiv.visualstudio.com/DevDiv/_queries/edit/951013/)

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: .NET Core 3.0 (WPF)

.NET Core SDK version:  3.0.100-preview8-013442

Windows Version: Windows10

Screen Reader Version: Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link.

Repro Steps:

1) Open Visual Studio.
2) Hit on File and select Open->Project/Solution.
3) Run exe file for "Message Box" from the build app folder provided.​
4) Main Window for "Message Box" solution will appear.
5) Start screen reader.
8) Navigate to any "Combo box" and Expand it using Alt + Down Arrow key.

Actual:

When we expand any Combo box, Screen reader is not announcing its state that Combo box is Expanded.

It is announcing its collapsed state properly. 

Note: Similar issue is observed with Chrome + NVDA.

Expected:

Screen reader should announce state of every combo box as it is collapsed or expanded properly. 

User Impact:

If proper state of the interactive element is not announced to the screen reader users then will not understand on activating the element some action has taken place for not. 

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 1.3.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by rladuca** [comment]
This is a Narrator bug, not a WPF bug.

NVDA reads this correctly.
